pkg/utils/utils.go

```diff
@@ -247,3 +250,37 @@ func WithLabels(labels map[string]string, existing map[string]string) map[string
 
 	return labels
 }
+
+// AddFinalizer adds finalizer to VM import CR
+func AddFinalizer(cr *v2vv1alpha1.VirtualMachineImport, name string) {
+	if HasFinalizer(cr, name) {
+		return
+	}
+	cr.Finalizers = append(cr.Finalizers, name)
```

Comment:
Is intentional to not persist the `Finalizers`? It will be changed only to runtime object, and once we re-fetch it, it won't be there.

Reply:
Thanks for finding, will make sure it is updated